js/backup_in.js

```diff
@@ -362,12 +390,18 @@ function backup_generate()
         var bip32_keychain = new BIP32(xpriv);
         var electrum_key_btc_leg = bip32_keychain.derive("m/44'/0'/0'");
         electrum_key_btc_leg = base58check.encode(new Buffer(electrum_key_btc_leg.extended_private_key, 'hex'));
+        var electrum_key_btc_leg_pub = bip32_keychain.derive("m/44'/0'/0'");
```

Comment:
why not drop this line and use `electrum_key_btc_leg.extended_public_key` in the following line?

Reply:
I tried that when writing this patch, but turns out once it is encoded in hex in 392 it can't be reused.

Reply:
that's fucked up